src/main/resources/openapi/event.yaml

```diff
@@ -535,6 +535,37 @@ paths:
             application/json+problem:
               schema:
                 $ref: './problem.yaml'
+  /admin/events/obfuscate:
+    post:
+      tags:
+        - Event
+      summary: Obfuscate event text
+      description: |-
+        Obfuscate event text
+      operationId: adminObfuscateEveByIds
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - eve_ids
+              properties:
+                eve_ids:
+                  minLength: 1
+                  type: array
+                  items:
+                    type: integer
+      responses:
+        '204':
```

Comment:
just noticed now that the LLD states response should be 200. If you believe we should be returning 204 instead, it would be good to raise it and get the LLD updated accordingly

Reply:
Good spot updated